### DISCUSSION TITLE
The gr3at cook thread

### DISCUSSION THREAD

**Discussion by ryanmurray** [nebhead/PiFire#79] [Show and tell]
Ned you said we should start a what we cooked thread so last night I put the new pifire to the test smoked a pork shoulder for 12 hours overnight woke up wrapped and finish burned it was tasty and the pi fire made me not have to check anything as far as walking to the grill 😊![IMG_20220205_130106.jpg](https://user-images.githubusercontent.com/1445752/152661305-f1766f40-eb91-41a7-a58e-8c4e3d452b2d.jpg)

**Comment by rwp120** [comment]
![image](https://user-images.githubusercontent.com/26973468/152664770-01775f2f-1f4d-41d7-8c3b-2f7b2533dd60.jpeg)

Great idea sharing cooks. I love a good pork butt but for my first long cook on the PiFire i went with an 11 pound brisket today. Grill temps swung wildly and got upwards of 350 multiple times but the brisket came out perfect (luckily). After the cook, I discovered that my chimney cap was way too high and the wind was drawing the heat out and it wasn't an issue with the new device. Can't wait to make brisket chili with the leftovers tomorrow.

**Reply by ryanmurray** [reply]
ive got a 16 pound brisket to do soon. Next weekend for valentines day i have a big prime rib to do. I have a traeger elite that looks like a bbq so i have no chimney.

**Reply by nebhead** [reply]
I only have the after-shot:
![image](https://user-images.githubusercontent.com/19876886/152688492-4039d667-9c3d-4739-8a7e-00560c4f981f.png)
From a couple of weeks ago.  This pulled pork was pure heaven.  Cooked with Pit Boss Apple Pellets (not typically my favorite), plenty of rub (I use Adkins Western Rub: http://www.adkinsbbq.com/index.html).  I started cooking this a little late (9AM) but started it around 250F until it reached 160 internal temp.  Wrapped, increased the cook temp to 275F and took it to 209F internal temp.  Two probes in this one was key.  Finished around 5PM, which ended up being perfect.  Made sandwiches the first night, with home-made bread.  Then tacos another night, and finally Lentil soup (yum!).

**Comment by ryanmurray** [comment]
Today was prime rib day smoked for 5 hours between 190 and 160 towards end I am full ![20220213_140459.jpg](https://user-images.githubusercontent.com/1445752/153790887-b4d39a75-3092-4d44-a6c8-b479fb189b13.jpg)![20220213_173843.jpg](https://user-images.githubusercontent.com/1445752/153790893-625abe53-4ca8-4f34-ad41-eadb2c8e3ad0.jpg)

**Reply by nebhead** [reply]
Wow, wow, wow!!  I am sooo jealous. That looks perfect.

**Reply by ryanmurray** [reply]
> Wow, wow, wow!!  I am sooo jealous. That looks perfect. 

My traeger is so much more enjoyable now that I don't have to go check it constantly
